server/server.go

```diff
@@ -144,15 +144,26 @@ func (server *Server[state]) handle(handler Handler[state]) http.HandlerFunc {
 	}
 }
 
-func (server *Server[state]) handleMiddleware(
+// Utility function that converts stdlib middleware to Apollo middleware
+func HandlerMiddlewareStd[state State](
```

Comment:
Ik vind die functienaam niet erg duidelijk. Zeker met de `HandlerMiddleware` functie hieronder die iets helemaal anders doe voor niet-stdlib middleware.

Reply:
Ik ook niet. Heb je een voorstel voor een betere naam? Ik had met gebaseerd op `http.HandlerFunc` in de stdlib, wat wel min of meer overeenkomt met de functie hieronder maar niet echt met deze aangezien deze er geen handler van maakt 😛  "ConvertMiddleware" ofzo misschien? Alleen zou je dat ook in beide richtingen kunnen opvatten...

Reply:
`ConvertToApolloMiddleware`? Dat is redelijk self explanatory I guess

Reply:
Aangepast!